src/inner_runtime.rs

```diff
@@ -131,7 +137,11 @@ pub struct InnerRuntime {
 }
 impl InnerRuntime {
     pub fn new(options: RuntimeOptions) -> Result<Self, Error> {
-        let loader = Rc::new(RustyLoader::new(options.module_cache));
+        let loader = Rc::new(RustyLoader::new(

```

Comment:
I might move these into a LoaderOptions or something eventually but that's not public facing so not a big deal